mass2/core/mass_add_lines_truebq.py

```diff
@@ -1,3 +1,7 @@
+"""
+Add Am241Q and Am243Q lines to the known spectral shapes, if not already present
```

Comment:
Add Am241Q and Am243Q lines to the known spectral shapes, you must import this file if you want these line shapes. It's not imported automatically.